Sources/ArcGISToolkit/Components/Offline/PreplannedMetadataView.swift

```diff
@@ -0,0 +1,118 @@
+                    HStack {
+                        Spacer()
+                        LoadableImageView(loadableImage: image)
+                            .clipShape(.rect(cornerRadius: 10))
+                            .padding(.vertical, 10)
+                        Spacer()
+                    }
+                }
+                VStack(alignment: .leading) {
+                    Text("Name")
+                        .font(.caption)
+                        .foregroundStyle(.secondary)
+                    Text(model.preplannedMapArea.title)
+                        .font(.subheadline)
+                }
+                VStack(alignment: .leading) {
+                    Text("Description")
+                        .font(.caption)
+                        .foregroundStyle(.secondary)
+                    if model.preplannedMapArea.description.isEmpty {
+                        Text("This area has no description.")
+                            .font(.subheadline)
+                            .foregroundStyle(.tertiary)
+                    } else {
+                        Text(model.preplannedMapArea.description)
+                            .font(.subheadline)
+                    }
+                }
+                VStack(alignment: .leading) {
+                    Text("Size")
+                        .font(.caption)
+                        .foregroundStyle(.secondary)
+                    Text(model.directorySize.formatted(.byteCount(style: .file, allowedUnits: [.kb, .mb])))
+                        .font(.subheadline)
+                }
+            }
+            Section {
+                HStack {
+                    Image(systemName: "trash.circle.fill")
+                        .symbolRenderingMode(.palette)
+                        .foregroundStyle(.red, .gray.opacity(0.1))
+                        .font(.title)
+                    Button("Delete Map Area", role: .destructive) {
+                        dismiss()
+                        model.removeDownloadedPreplannedMapArea()
+                        Task {
+                            await model.load()
+                        }
+                    }
+                }
+            }
+        }
+        .navigationTitle(model.preplannedMapArea.title)
+        .navigationBarTitleDisplayMode(.inline)
+        .navigationBarBackButtonHidden()
+        .toolbar {
+            ToolbarItem(placement: .confirmationAction) {
+                Button {
+                    dismiss()
+                } label: {
+                    Text("Done")
+                        .bold()
+                }
+            }
+        }
+    }
+}
+
+#Preview {
+    struct MockPreplannedMapArea: PreplannedMapAreaProtocol {
+        var packagingStatus: PreplannedMapArea.PackagingStatus? = .complete
+        var title: String = "Mock Preplanned Map Area"
+        var description: String = "This is the description text"
+        var thumbnail: LoadableImage? = nil
```

Comment:
It doesn't make a huge difference, but since these are all constants, they should just be computed.

```suggestion
        var packagingStatus: PreplannedMapArea.PackagingStatus? { .complete }
        var title: String { "Mock Preplanned Map Area" }
        var description: String { "This is the description text" }
        var thumbnail: LoadableImage? { nil }
```